src/components/MovieBooking/MovieSeats.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect, useState } from "react";
+import MovieModal from "./MovieModal";
+import whiteSeatSvg from "../Assets/whiteSeat.svg";
+import blueSeatSvg from "../Assets/blueSeat.svg";
+import blackSeatSvg from "../Assets/BlackSeat.svg";
+import {
+  SeatStyle,
+  SeatTableStyle,
+  SeatButtonStyle,
+  ModelScreenStyle,
+  DigitStyle,
+  SeatRowStyle,
+} from "../Styles/Seat.style";
+
+const digitArr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+const seats = {
+  data: [
+    {
+      id: "A",
+      seat: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+    },
+    {
+      id: "B",
+      seat: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+    },
+    {
+      id: "C",
+      seat: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+    },
+    {
+      id: "D",
+      seat: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+    },
+  ],
+};
```

Comment:
Refactor this to reduce code repetition.  You can make use of digitArr in place of seat values instead of repeating same array again and again. This can be extracted to data.js file in the same folder.